ksql-functional-tests/src/test/resources/query-validation-tests/session-windows.json

```diff
@@ -99,32 +99,12 @@
         {"topic": "test_topic", "key": 1, "value": "1,100,200", "timestamp": 40000}
       ],
       "outputs": [
-        {"topic": "S2", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": null, "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": "0,5", "timestamp": 10000, "window": {"start": 0, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": "1,100", "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": null, "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": "1,200", "timestamp": 40000, "window": {"start": 10000, "end": 70000, "type": "time"}}
-      ]
-    },
-    {
-      "name": "import session stream and group by",
```

Comment:
This test is incorrect since at the moment kstreams does not support reading windowed key and aggregating on it!